### PR TITLE
Fix mobile authentication panel and custom elements naming

### PR DESCRIPTION
**Pull request by llienher**

### CODE REVIEW

**Comment by sbrunner** [comment]
Shouldn't be the inverse?
auth/PanelElement => gmf-auth-panel
Then 
auth/FormElement => gmf-auth-form, not gmf-auth-component
?

**Comment by llienher** [comment]
We had this change with the canvas PR, was it an error ?
It makes more sense to me that we use gmf-auth-form by the way. :-)

File: contribs/gmf/apps/desktop_alt/index.html.ejs [hunk 1]
```diff
@@ -96,15 +96,14 @@
...
       <!-- Filter -->
-      <gmf-button-filter slot="tool-button">
+      <gmf-filter-button slot="tool-button">
         <button ngeo-btn class="btn btn-default" ng-model="mainCtrl.filterSelectorActive"
         data-toggle="tooltip" data-placement="left" data-original-title="{{'Filter'|translate}}">
         <span
           class="fa"
           ng-class="mainCtrl.gmfDataSourceBeingFiltered.dataSource && mainCtrl.gmfDataSourceBeingFiltered.dataSource.filterRules ? 'fa-funnel-dollar' : 'fa-filter'"></span>
       </button>
-
```
Comment (sbrunner) [inline contribs/gmf/apps/desktop_alt/index.html.ejs:106]: Keep space :-)

**Review by sbrunner — APPROVED** [review]
Thanks :-)

**Review by sbrunner — APPROVED** [review]

**Review by ger-benjamin — APPROVED** [review]

**Pull request merged**